draft-irtf-cfrg-frost.md

```diff
@@ -990,9 +983,9 @@ The value of the contextString parameter is "FROST-ED25519-SHA512-v11".
 Normally H2 would also include a domain separator, but for compatibility with {{!RFC8032}}, it is omitted.
 
 Signature verification is as specified in {{Section 5.1.7 of RFC8032}} with the
-constraint that implementations MUST check the group equation [8][z]B = [8]R + [8][c]PK
+constraint that implementations MUST check the group equation `[8][z]B = [8]R + [8][c]PK`
```

Comment:
The brackets were being interpreted as references. We could escape them, but this seems easier and better looking